docs/samm.md

```diff
@@ -0,0 +1,220 @@
+# Vspec - Semantic Aspect Meta Model (SAMM) exporter
```

Comment:
Great addition - now you have a better documentation that most other tools :-)

Reply:
Thank you for the hint on this :)